### PR TITLE
MSTR-2: Fix transactions total

### PR DESCRIPTION
**Pull request by masmerino13**
Stop to include failed values

### CODE REVIEW

File: src/apps/myaccount/submodules/transactions/transactions.js [hunk 1]
```diff
@@ -150,9 +150,14 @@ define(function(require) {
...
 					args.callback({
-						amount: _.reduce(formatted, function(acc, transaction) {
-							return acc + parseFloat(transaction.amount);
-						}, 0),
+						amount: _
+							.chain(formatted)
+							.filter(function(transaction) {
+								return transaction.isApproved;
+							})
+							.reduce(function(acc, transaction) {
```
[the remark this answers is not in the document]
Comment (masmerino13) [reply src/apps/myaccount/submodules/transactions/transactions.js]: cool, thanks!

File: src/apps/myaccount/submodules/transactions/transactions.js [hunk 2]
```diff
@@ -150,9 +150,14 @@ define(function(require) {
 						.value();
 
 					args.callback({
-						amount: _.reduce(formatted, function(acc, transaction) {
-							return acc + parseFloat(transaction.amount);
-						}, 0),
+						amount: _
+							.chain(formatted)
+							.filter(function(transaction) {
```
[the remark this answers is not in the document]
Comment (masmerino13) [reply src/apps/myaccount/submodules/transactions/transactions.js]: Really nice, thanks!

File: src/apps/myaccount/submodules/transactions/transactions.js [hunk 3]
```diff
@@ -150,9 +150,10 @@ define(function(require) {
 						.value();
 
 					args.callback({
-						amount: _.reduce(formatted, function(acc, transaction) {
-							return acc + parseFloat(transaction.amount);
-						}, 0),
+						amount: _
+							.chain(formatted)
+							.filter('isApproved')
+							.sumBy('amount'),
```
Comment (azefiel) [inline src/apps/myaccount/submodules/transactions/transactions.js]: Although it works in this particular case (thanks to the [type casting power](https://github.com/2600hz/monster-ui/blob/master/src/js/lib/monster.util.js#L954) of `monster.util#formatPrice()`), the result of a chain sequence should always be unwrapped using `_#value` to retrieve the actual value, or else you will end up with the lodash wrapper instance.
```suggestion
							.sumBy('amount')
							.value(),
```

Comment (masmerino13) [reply src/apps/myaccount/submodules/transactions/transactions.js]: oups, thanks

**Pull request merged**